tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,33 @@
+    testScenario(this.server);
+    await visit('/catalog');
+    await animationsSettled();
+
+    await this.pauseTest();
+    assert.ok(helpscoutBeacon.isPresent, 'Helpscout Beacon is present');
+    await this.pauseTest();
+  });
+
+  test("beacon doesn't render on concepts page", async function (assert) {
+    testScenario(this.server);
+    await conceptsPage.visit();
```

ESLint

[error] 'conceptsPage' is not defined.

(no-undef)